ksql-workshop/ksql-workshop.adoc

```diff
@@ -0,0 +1,885 @@
+= Workshop: Real-time SQL Stream Processing at Scale with Apache Kafka and KSQL
+:source-highlighter: pygments
+:doctype: book
+:sectnums:
+Robin Moffatt <@rmoff>
+v0.10, 19 Jun 2018
+
+:toc:
+
+== Introduction
+
+This is a two-hour hands-on workshop.
+
+[quote]
+____
+KSQL is the streaming SQL engine for Apache Kafka. Using just SQL it is possible for developers to build powerful stream processing applications. This talk will show practical examples of KSQL:
+
+* Filtering streams of data
+* Joining live events with reference data (e.g. from a database)
+* Stateful aggregations
+* Convert streams from JSON to AVRO
+____
+
+image::images/ksql_workshop_01.png[Diagram of what's being built with the KSQL demo]
+
+Don't forget to check out the #ksql channel on our https://slackpass.io/confluentcommunity[Community Slack group]
+
+——_Robin Moffatt https://twitter.com/rmoff/[@rmoff], 19 Jun 2018_
+
+== Pre-requisites
+
+* Docker
+* Docker Compose
+* 8GB+ RAM
+
+IMPORTANT: *THIS MUST BE DONE _BEFORE_ THE WORKSHOP - The docker images are large and will take time to download. Please don't wait until the workshop as the wi-fi will not cope!*
+
+[IMPORTANT]
+====
+
+0. In Docker’s advanced settings, increase the memory dedicated to Docker to at least 8GB.
+** Don't forget to do this
+
+1. Clone the workshop repo to your local machine:
++
+[source,bash]
+----
+git clone https://github.com/confluentinc/demo-scene.git
+----
+
+0. Did you remember to increase the memory dedicated to Docker to at least 8GB?
+** You really do need to do this
+
+2. Change working directory
++
+[source,bash]
+----
+cd ksql-workshop
```

Comment:
this will need to become `cd quickstart-demos/ksql-workshop`

Reply:
Great catch. I updated the adoc.